modules/node-shared/src/main/scala/org/tessellation/node/shared/config/types.scala

```diff
@@ -126,4 +127,11 @@ object types {
     maxSize: PosInt
   )
 
+  case class DoubleSignDetectConfig(
+    minDistance: PosInt,
```

Comment:
what does it mean? is it a min distance to start the detection, max distance after which double signs are ignored?

Reply:
It is the maximum amount of "time" that two snapshots with the same ordinal, but with different hashes, can be spaced apart and still be considered a Double Signing event. It has been renamed to `maxDetectionDelta`.